lib/loc/loc.go

```diff
@@ -238,6 +234,24 @@ func Deduplicate(ls []Locator) (result []Locator) {
 	return result
 }
 
+// GetUpdatedDependencies compares installedDeps against the updateDeps
+// and returns only locators from updateDeps that are updates of those given with installedDeps
+func GetUpdatedDependencies(installedDeps, updateDeps []Locator) ([]Locator, error) {
```

Comment:
Moved from lib/app from what used to be [GetUpdatedDependencies](https://github.com/gravitational/gravity/blob/1d5bd035e9cddefafaf5d9f1471f68e9589a3d45/lib/app/apps.go#L141)